src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/SdkTemplateTests.cs

```diff
@@ -72,6 +72,112 @@ public void VerifyMSTestTemplate(DotNetLanguage language)
             DotNetSdkActions.Test);
         newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
     }
+    
+    [Theory]
+    [InlineData(DotNetLanguage.CSharp)]
+    public void VerifyCSharpWPFTemplate(DotNetLanguage language)
+    {
+        var newTest = new SdkTemplateTest(
+            nameof(SdkTemplateTest), language, _scenarioTestInput.TargetRid, DotNetSdkTemplate.Wpf,
+            DotNetSdkActions.Test | DotNetSdkActions.Publish);
+        newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
+    }
+    
+    [Theory]
+    [InlineData(DotNetLanguage.VB)]
+    public void VerifyVBWPFTemplate(DotNetLanguage language)
+    {
+        var newTest = new SdkTemplateTest(
+            nameof(SdkTemplateTest), language, _scenarioTestInput.TargetRid, DotNetSdkTemplate.Wpf,
+            DotNetSdkActions.Test | DotNetSdkActions.Publish);
+        newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
+    }
+    
+    [Theory]
+    [InlineData(DotNetLanguage.CSharp)]
+    public void VerifyWebAppTemplate(DotNetLanguage language)
+    {
+        var newTest = new SdkTemplateTest(
+            nameof(SdkTemplateTest), language, _scenarioTestInput.TargetRid, DotNetSdkTemplate.WebApp,
+            DotNetSdkActions.Test | DotNetSdkActions.PublishComplex);
+        newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
+    }
+    
+    
+    [Theory]
+    [InlineData(DotNetLanguage.CSharp)]
+    public void VerifyCSharpWinformsTemplate(DotNetLanguage language)
```

Comment:
Same comment as https://github.com/dotnet/scenario-tests/pull/11/files#r1289128395 for the winforms templates.

Reply:
This issue has been resolved with multiple InlineData attributes